datasets/us/ss_wanted/us_ss_wanted.yml

```
@@ -0,0 +1,53 @@
+title: "United States Secret Service Most Wanted Fugitives"
+entry_point: crawler.py
+prefix: us-ss-wanted
```

Is the length of the prefix ok in this case? @jbothma

---

yeah it's fine. I've proposed a shorter one.

```suggestion
prefix: us-sswant
```